config/src/config/state_sync_config.rs

```diff
@@ -128,7 +128,7 @@ pub struct StateSyncDriverConfig {
 impl Default for StateSyncDriverConfig {
     fn default() -> Self {
         Self {
-            bootstrapping_mode: BootstrappingMode::DownloadLatestStates,
+            bootstrapping_mode: BootstrappingMode::ExecuteOrApplyFromGenesis,
```

Comment:
Is this required, or was it just to make the hack easier? 😄

Reply:
this is only to bypass the config sanitizer, otherwise it just panics the tests